Hardware/Hardware/CardReader/CardReaderService.swift

```diff
@@ -10,6 +10,10 @@ public protocol CardReaderService {
     /// The Publisher that emits the connected readers
     var connectedReaders: AnyPublisher<[CardReader], Never> { get }
 
+    /// The Publisher that emits the list of known readers. Known readers are readers that the merchant has
+    /// connected to in the past.
+    var knownReaders: AnyPublisher<[CardReader], Never> { get }
```

Comment:
So, there are things in `CardReader` that might not make sense to persist, like battery level. But there are a few things that are interesting. Since we are not adding or removing known readers in this PR, that's ok. A future PR should be able to safety chose a different structure here with minimal impact to the subscribing code.

Reply:
> Known readers are readers that the merchant has connected to in the past.

How do we know the merchant has already connected to a particular reader?

Reply:
Good point. I've captured this in https://github.com/woocommerce/woocommerce-ios/issues/4047#issuecomment-828580028 and we'll move this concept to the store instead